setup.py

```diff
@@ -44,19 +70,30 @@ def get_version(module='spyder_kernels'):
 setup(
     name='spyder-kernels',
     version=get_version(),
-    keywords='spyder jupyter kernel',
+    keywords='spyder jupyter kernel ipython console shell',
     url='https://github.com/spyder-ide/spyder-kernels',
+    download_url="https://www.spyder-ide.org/#fh5co-download",
     license='MIT',
     author='Spyder Development Team',
-    description='Jupyter kernels for the Spyder console',
+    author_email="spyderlib@googlegroups.com",
+    description="Jupyter kernels for the Spyder IDE's IPython consoles.",
+    long_description="\n".join(DOCLINES[4:]),
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     install_requires=REQUIREMENTS,
     include_package_data=True,
+    platforms=["any"],
     classifiers=['Development Status :: 5 - Production/Stable',
                  'Framework :: Jupyter',
+                 'Framework :: IPython',
                  'Intended Audience :: Developers',
                  'License :: OSI Approved :: MIT License',
                  'Operating System :: OS Independent',
+                 'Programming Language :: Python :: 2',
                  'Programming Language :: Python :: 2.7',
-                 'Programming Language :: Python :: 3']
+                 'Programming Language :: Python :: 3',
+                 'Programming Language :: Python :: 3.4',
+                 'Programming Language :: Python :: 3.5',
+                 'Programming Language :: Python :: 3.6',
+                 'Programming Language :: Python :: 3.7',
```

Comment:
I don't want to keep updating this list after each Python release. So please leave it was it was:

    'Programming Language :: Python :: 2.7',
    'Programming Language :: Python :: 3'

I borrowed this from the IPython/Jupyter guys, by the way.

Reply:
Okay; I was referencing the SciPy setup.py, incidentally (among others). 

I mean, its not *that* bad to update it (or not, as appropriate) once every one or two years when we add or drop compatibility with a Python version, since it has an actual purpose: to make it clearer to users browsing PyPI what we've compatible with (since we don't support all Python 3 versions, just >=3.4, and its not impossible we might temporarily be incompatible with a new release). Its also now inconsistent with the main Spyder setup.py since you approved these same changes there. But I reverted it back for now...